instrumentation/src/javatest/org/oppia/android/instrumentation/ExplorationPlayerTest.kt

```diff
@@ -0,0 +1,108 @@
+package org.oppia.android.instrumentation
+
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.UiCollection
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiScrollable
+import androidx.test.uiautomator.UiSelector
+import androidx.test.uiautomator.Until
+import org.junit.Assert.assertNotNull
+import org.junit.Before
+import org.junit.Test
+
+/** Tests for Explorations. */
+class ExplorationPlayerTest {
+  private val OPPIA_PACKAGE = "org.oppia.android"
+  private val LAUNCH_TIMEOUT = 30000L
+  private val TRANSITION_TIMEOUT = 5000L
+  private lateinit var device: UiDevice
+
+  @Before
+  fun startOppiaFromHomeScreen() {
+    // Initialize UiDevice instance
+    device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
```

Comment:
import the required function only not the whole class `UiDevice.`

Reply:
verified that we need to entire class for UiDevice.

but for Until we can only import Until.hasObject

but `Until.hasObject` seems more readable than `hasObject`

shall I only use the function import here?

Reply:
Yes, let's go with the hasObject, import whole Until.hasObject

Reply:
removed it